src/components/InputForm.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components';
+import plusImg from '../assets/img/plus.svg';
+
+const InputStyle = styled.input`
+  background-color: #d9d9d9;
+  border: none;
+  outline: none;
+  border-radius: 10px;
+  padding: 5px 5px 5px 30px; /*텍스트와 이미지 겹침 방지를 위한 패딩 조정*/
+  width: 100%;
+  line-height: 20px;
+  cursor: text;
+
+  background-image: url('${plusImg}');
+  background-position: 5px center; /*이미지 위치 조정*/
+  background-repeat: no-repeat;
+  background-size: 20px 20px;
+`;
+
+export default function InputForm({ value, setValue, onAddToDo }) {
```

Comment:
props로 넘겨받을때 구조분해할당하신거 너무 좋아요! 저의 경우에는 2개 이상의 props인 경우 가독성을 위해 아래와 같이 쓰는데요, 기능은 같으니까 참고만하시면 될 것 같아요 ㅎㅎ
```suggestion
export default function InputForm(props) {
    const { value, setValue, onAddToDo } = props;
```

Reply:
여러 개 넘겨 받을 때 민영님 코드처럼 작성하면 가독성이 더 좋을 것 같아요! 감사합니다 👍